lotto/src/main/kotlin/lotto/domain/Lotto/Lotto.kt

```diff
@@ -0,0 +1,46 @@
+package lotto.domain.Lotto
+
+import lotto.domain.result.JudgeResult
+
+class Lotto(
+    val numbers: Set<Number>
+) {
+
+    fun judgeWinning(winningLotto: Lotto, bonusNumber: Number): JudgeResult {
+        var sameCount = 0
+        var bonusWin = false
+
+        for (number in numbers) {
+            if (winningLotto.contains(number)) {
+                sameCount += 1
+            }
+        }
+
+        if (numbers.contains(bonusNumber)) {
+            bonusWin = true
+        }
+
+        return JudgeResult(sameCount, bonusWin)
+    }
+
+    private fun contains(number: Number) =
+        numbers.contains(number)
```

Comment:
```suggestion
    private fun contains(number: Number) = numbers.contains(number)
```
inline으로 써도 되지 않을까요?

Reply:
넵! 요거는 팀 컨벤션이라 익숙해졌는데 개인적 취향이라 봐주십셔!